lib/puppet/type/firewalld_direct_purge.rb

```diff
@@ -1,8 +1,8 @@
 require 'puppet'
 require 'puppet/parameter/boolean'
-require 'puppet/type/firewalld_direct_chain'
-require 'puppet/type/firewalld_direct_rule'
-require 'puppet/type/firewalld_direct_passthrough'
+require File.dirname(__FILE__).concat('/firewalld_direct_chain.rb')
+require File.dirname(__FILE__).concat('/firewalld_direct_rule.rb')
+require File.dirname(__FILE__).concat('/firewalld_direct_passthrough.rb')
 
```

Comment:
@vilkasx Just for consistency, could you construct the filename in the same way as the providers do (see #79 )

eg: 

```ruby
File.join(File.dirname(__FILE__),  'firewalld_direct_chain.rb')
```

It's a bit nicer than using concat.....